src/app/static/src/tests/filteredData/getters.test.ts

```diff
@@ -175,105 +158,6 @@ describe("FilteredData getters", () => {
         expect(filters).toBeNull();
     });
 
-    it("gets unfilteredData when selectedDataType is Survey", () => {
```

Comment:
These tests moved to own file https://github.com/mrc-ide/hint/pull/170/files#diff-62db21a33d4682b884efe0101fc4ef1e